src/main/scala/fs/package.scala

```diff
@@ -19,14 +19,24 @@ package object fs {
   type XattrFiller       = net.fusejna.XattrFiller
   type XattrListFiller   = net.fusejna.XattrListFiller
 
+  trait PathResolving {
+    def resolvePath: String => jio.Path
+  }
```

Comment:
Does this warrant being broken out independently at this point? Does it have some immediately valuable use other than as part of a FuseFilesystem? For me types like `FuseFilesystem with PathResolving` are a last resort, and if I write such a type more than once I'm going to look for reasons why they should be part of the same thing.